xrpl/client/jsonrpc/jsonrpc_client.go

```diff
@@ -117,6 +118,12 @@ func (c *JsonRpcClient) SendRequest(reqParams client.XRPLRequest) (client.XRPLRe
 	return &jr, nil
 }
 
+func (c *JsonRpcClient) Submit(tx interface{}, failHard bool) (client.XRPLResponse, error) {
```

Comment:
To make it simpler, I would rename it `SubmitTransaction` and instead of taking a tx as a param, just pass the tx_blob 👍🏻 

Or it is a required feature to submit a transaction?

Reply:
Maybe we can start with a `SubmitTransaction` function accepting a `tx_blob` only for now.
We can iterate and upgrade the function in a later PR.